### PR TITLE
Some typo fixes, and paragraph shortening.

### PR DESCRIPTION
**Pull request by ilopezgp**
- Fixes several typos of the manuscript submission to JOSS: https://github.com/openjournals/joss-reviews/issues/4869,
- Adds missing reference,
- Shortens section without modifying content,
- Updates pdf.

### CODE REVIEW

File: paper.bib [hunk 1]
```diff
@@ -37,6 +37,20 @@ @article{julia
   journal   = {{SIAM} Review}
 }
 
+@inproceedings{ge2018t,
```
Comment (odunbar) [inline paper.bib:40]: is there a doi for this? or is this biburl? I know the JOSS editor runs a DOI check.

**Review by odunbar — APPROVED** [review]
LGTM - note the citation data may flag up the JOSS DOI check. The editor may need to know this

File: paper.bib (hunk 1 above)
Comment (ilopezgp) [reply paper.bib:40]: I checked and they do not have a doi, unfortunately. Just the URL, which should be "permanent" to some extent

**Pull request merged**